From fdee05b5abfc54b0c1de8326786a39d4cab58afa Mon Sep 17 00:00:00 2001
From: Nishanth Aravamudan <nish.aravamudan@canonical.com>
Date: Wed, 2 Nov 2016 23:33:37 +0100
Subject: Update for sphinx internal structure change layout

---
 doc/build/templates/genindex.mako | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/build/templates/genindex.mako b/doc/build/templates/genindex.mako
index 0a25b67..2f45f79 100644
--- a/doc/build/templates/genindex.mako
+++ b/doc/build/templates/genindex.mako
@@ -21,7 +21,7 @@
         numcols = 1
         numitems = 0
     %>
-% for entryname, (links, subitems) in entries:
+% for entryname, (links, subitems, dummy) in entries:
 
 <dt>
     % if links:
